atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -93,7 +93,6 @@ private AtlasDbConstants() {
 
     public static final int DEFAULT_TABLES_TO_PUBLISH_TABLE_LEVEL_METRICS = 10;
 
-    public static final long STARTING_TS = 1L;
```

Comment:
note that this will break internal backup service. I think it's not necessarily terrible as the impact is small. this should be compile time visible (i.e. it's not actually used in any atlasdb _libraries_ as far as I can tell) and there is pretty much a drop in replacement.